easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -14,15 +14,26 @@ toolchainopts = {'usempi': True}
 # Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
 sources = [SOURCELOWER_TAR_GZ]
 
+patches = ['patch.5.4.1.08072015', 'patch.5.4.1.27082015', 'patch.5.4.1.06112015']
```

Comment:
@migueldiascosta I just checked, the name of the patch file doesn't matter, it will get applied regardless.

But apparently the easyconfig unit tests only check for the presence of patch files ending in `.patch`, that should be fixed; patch files are expected to be included with the easyconfig files.

Even though VASP is licenses, I think that should be doable here too, since these patches are available publicly at http://cms.mpi.univie.ac.at/wiki/index.php/Installing_VASP#Patches.

Although, I'm not sure if we're allowed to copy/redistribute them, however...